### PR TITLE
Upgrading play24

### PR DESCRIPTION
**Pull request by muuki88**

### CODE REVIEW

File: conf/routes [hunk 1]
```diff
@@ -27,3 +27,4 @@ GET     /authuser               controllers.Users.authUser()
 
 # Map static resources from the /public folder to the /assets URL path
 GET     /assets/*file           controllers.Assets.versioned(path="/public", file: Asset)
+GET     /webjars/*file          controllers.WebJarAssets.at(file)
```
Comment (mariussoutier) [inline conf/routes]: Is this required in Play 2.4?

Comment (muuki88) [inline conf/routes]: Ah, no. Was playing around with font-awesome

**Comment by mariussoutier** [comment]
I'll close this one for the moment. Upgrade to Play 2.4 should also use DI, ideally Cake.

**Comment by mariussoutier** [comment]
FYI I have WIP branch locally, will publish soon.

**Pull request closed without merging**